cmd/kuberhealthy/kuberhealthy.go

```diff
@@ -1151,6 +1151,7 @@ func (k *Kuberhealthy) healthCheckHandler(w http.ResponseWriter, r *http.Request
 // this will return the state of ALL found checks.
 // Failures to fetch CRD state return an error.
 func (k *Kuberhealthy) getCurrentState(namespaces []string) health.State {
+	log.Debugln("getCurrentState: ", namespaces)
```

Comment:
is this intentional?

Reply:
this was not...perhaps Eric and I wrote this when we were debugging something earlier...have no idea how that got in there